dlc_finance/office1.lua

```diff
@@ -116,8 +116,8 @@ FinanceOffice1 = {
             elseif doorSide:lower() == "right" then
                 FinanceOffice1.Safe.isRightDoorOpen = true
             else
-                Citizen.Trace("[bob74_ipl] Warning: " .. doorSide .. " is not a correct value. Valid values are:")
-                Citizen.Trace("left right")
+                print("[bob74_ipl] Warning: " .. doorSide .. " is not a correct value. Valid values are:")
+                print("left right")
```

Comment:
Replacing it here has a different effect since `Trace` doesn't add a newline at the end.

Reply:
Combined:

                print("[bob74_ipl] Warning: " .. doorSide .. " is not a correct value. Valid values are: left right")